wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/AjaxDownloadBehavior.java

```diff
@@ -323,10 +323,30 @@ private String getName()
 	}
 
 	/**
+	 * Controls if anti cache parameter should be added to the URL or not
+	 *
+	 * @return {@code true} to add the anti cache request parameter, {@code false} - otherwise
+	 */
+	protected boolean shouldAddAntiCacheParameter() {
```

Comment:
What is this good for? The resource can control caching - I don't see the necessity for an antiCache parameter (as ResourceLink doesn't need have it either).

Reply:
I'll double-check (modify my tests) and remove it
I thought it would save some lines of code if anti-cache will be on by default